examples/foomedical/src/pages/account/MembershipAndBilling.tsx

```diff
@@ -1,14 +1,41 @@
-import { Box, Stack, Title } from '@mantine/core';
-import { getReferenceString } from '@medplum/core';
-import { Patient } from '@medplum/fhirtypes';
+import { Box, Stack, Table, Title } from '@mantine/core';
+import { formatCoding, getReferenceString } from '@medplum/core';
+import { Coverage, Patient } from '@medplum/fhirtypes';
 import { useMedplum } from '@medplum/react';
 import { InfoButton } from '../../components/InfoButton';
 import { InfoSection } from '../../components/InfoSection';
 
+function CoverageTable({ coverages }: { coverages: Coverage[] }): JSX.Element {
+  return (
+    <Table>
+      <Table.Thead>
+        <Table.Tr>
+          <Table.Th>Payor Name</Table.Th>
+          <Table.Th>Subscriber ID</Table.Th>
+          <Table.Th>Relationship to Subscriber</Table.Th>
+        </Table.Tr>
+      </Table.Thead>
+      <Table.Tbody>
+        {coverages.map((c) => (
+          <Table.Tr key={c.id}>
+            <Table.Td>{c.payor?.[0].display}</Table.Td>
+            <Table.Td>{c.subscriberId || '-'}</Table.Td>
+            <Table.Td>{formatCoding(c.relationship?.coding?.[0]) || '-'}</Table.Td>
```

Comment:
Note: I'm using `||` here because `??` does not work with `formatCoding` since it returns an empty string as default.